package.json

```diff
@@ -6,6 +6,10 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "dotenv": "^8.2.0",
+    "react": "^16.13.1",
```

Comment:
I dont think we need react, react-dom,and react-scripts as these are already added below.

Reply:
deleted duplicates